hackathon_site/dashboard/frontend/src/components/teamDetail/TeamCheckedOutOrderTable/TeamCheckedOutOrderTable.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
```

Comment:
something's off here, could you merge with develop?

Reply:
These changes shouldn't be showing up

Reply:
I did a git pull --rebase with develop and this happened...